.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: CI python selenium
+on:
+  pull_request:
+    branches:
+      - sdk
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
```

Comment:
Do we need to run on so many python version? Can we skip some where there are no major breaking changes?